kasa/aestransport.py

```diff
@@ -83,34 +86,32 @@ def hash_credentials(self, login_v2):
         """Hash the credentials."""
         if login_v2:
             un = base64.b64encode(
-                _sha1(self._credentials.username.encode()).encode()
+                _sha1(self.credentials.username.encode()).encode()
```

Comment:
Please refrain renames / private-public scope changes in this PR as it makes the diff much larger for no apparent reason, we can clean up as wanted afterwards in a separate PR that does not involve other changes.

Reply:
The problem was they got mixed in the last PR so the base classes had no underscore for the same variables.  I think we need to make them consistent one way or another.

Reply:
Ohh, I see.. But shouldn't we then rather change the base classes to have an underscore? :-) If need to be for some specific cases (if really, really necessary), we can expose them later on by defining a `@property`.